src/containers/settings/EditServiceScreen.tsx

```diff
@@ -188,50 +188,52 @@ class EditServiceScreen extends Component<IProps> {
         },
         isEnabled: {
           label: intl.formatMessage(messages.enableService),
-          value: service?.isEnabled,
-          default: DEFAULT_SERVICE_SETTINGS.isEnabled,
+          value: service?.isEnabled || DEFAULT_SERVICE_SETTINGS.isEnabled,
```

Comment:
This will also evaluate the 2nd condition if the 1st one returns a false. But, I think you wanted to only handle the `undefined` condition, am i right?

Reply:
@cino - could you please verify that my later commit into your PR branch - whether you think that's a better way to handle this issue?

Reply:
@cino - I am suggesting to user default value and the value seperatly where if value is not available better we use null assigned to value

Eg:

     isEnabled:{
          label: intl.formatMessage(messages.enableService),
          value: service?.isEnabled,
          default: DEFAULT_SERVICE_SETTINGS.isEnabled, // let it be
          value: service?.isEnabled || <we need to find and sent a value that can't be used as value  example undefined or null>,